services/database/migrations/20231109160023-sourcify.js

```diff
@@ -79,13 +79,12 @@ exports.up = function (db, callback) {
       db.runSql.bind(
         db,
         `CREATE TABLE verification_jobs (
-            id BIGSERIAL NOT NULL,
-            status varchar NOT NULL,
+            id uuid NOT NULL DEFAULT gen_random_uuid(),
             started_at timestamptz NOT NULL DEFAULT NOW(),
```

Comment:
As Kaan [pointed out here ](https://github.com/ethereum/sourcify/pull/1902#issuecomment-2653664891) I think we missed the `verification_endpoint` field.

Reply:
In general what we store in the db it's not what we return in the API, we actually have the `verification_endpoint` in the db, do we want to return it in the API? 

@kuzdogan

Reply:
The `verification_endpoint` field is not missing the SQL. See line 90.

I didn't include it in the API response, because I saw it as something for our internal statistical use. But I don't mind adding it to the API. If you see a benefit of adding it, let me know.

Reply:
I don't think that needs to be in the API. It's more for debugging purposes